src/components/Hero/Hero.tsx

```diff
@@ -32,11 +34,8 @@ const Hero = () => {
       <div className="mx-auto my-64 relative max-w-screen-md">
         <hr className={`w-20 absolute right-1 -top-10 ${css.line}`} />
         <ImageHero />
-        <p
-          className={`mt-5 text-base lg:text-3xl opacity-0`}
-          ref={textHeroSubtitleRef}
-        >
-          A maior comunidade do norte do Brasil
+        <p className={`mt-5 text-base lg:text-3xl opacity-0`} ref={textHeroSubtitleRef}>
```

Comment:
Aqui uma outra sugestão seria voltar pra 

`<p className="mt-5 text-base lg:text-3xl opacity-0" ref={textHeroSubtitleRef}>`

Reply:
Boa, alterei e aproveitei pra alterar o css ai do `css.line` que tava sem apply tb de tailwind, quando for mexendo no projeto vou vendo outros lugares que tao sem e vou ajustando